### PR TITLE
Publish to reindex task counts topic

### PR DESCRIPTION
**Pull request by RMPall**
### What

This pr attempts to publish the completed tasks to reindex task count topic. 

### How to review

Please check if the changes make sense.

### Who can review

Anyone except me

### CODE REVIEW

File: event/producer_test.go [hunk 1]
```diff
@@ -84,3 +93,51 @@ func TestProducer_ContentUpdated(t *testing.T) {
...
+			So(err, ShouldBeNil)
+
+			var avroBytes []byte
+			var testTimeout = time.Second * 5
+			select {
+			case avroBytes = <-pChannels.Output:
+				t.Log("avro byte sent to producer output")
+			case <-time.After(testTimeout):
+				t.Fatalf("failing test due to timing out after %v seconds", testTimeout)
+				t.FailNow()
```
Comment (nshumoogum) [inline event/producer_test.go:132]: No need for FailNow() after calling Fatalf(...) as Fatalf is equivalent to Logf followed by FailNow

File: service/service.go [hunk 2]
```diff
@@ -56,17 +56,30 @@ func Run(ctx context.Context, cfg *config.Config, serviceList *ExternalServiceLi
 		return nil, err
 	}
 
+	// Get Kafka producer for reindex task counts
+	producerForReindexTaskCounts, err := serviceList.GetKafkaProducerForReindexTaskCounts(ctx, cfg)
```
Comment (nshumoogum) [inline service/service.go:60]: So we need to extend the registered checkers to include new kafka producers and arguably Zebedee health check

And also update the graceful shutdown to stop writing to the kafka topics (close them down)

**Review by nshumoogum — CHANGES_REQUESTED** [review]

File: event/producer_test.go (hunk 1 above)
Comment (RMPall) [reply event/producer_test.go]: Fixed.

File: service/service.go (hunk 2 above)
Comment (RMPall) [reply service/service.go:60]: Fixed.

**Pull request merged**